docs/schemas/resource/IfcRepresentationResource/Entities/IfcRigidOperation.md

```diff
@@ -1,6 +1,10 @@
 # IfcRigidOperation
 
-A rigid operation specifies an offset in the coordinate reference system. It does not specify any conversion or distortion only a translation. The coordinate axes of the _IfcGeometricRepresentationContext_ can rotate and set a new basis coordinate system.
+A rigid operation specifies an offset in the coordinate reference system. It does not specify any conversion or distortion. It is a coordinate operation that tells that the whole virtual model is **translated** in the same way in the (x,y,z) plane.
+
+> EXAMPLE  If data is in truncated map coordinates (i.e., the map coordinates have the leading digits removed in x and y), then _IfcRigidOperation_ simply translates the data in x and y to replace the ignored leading digits. This is also known as subtraction.
+
+==The coordinate axes of the _IfcGeometricRepresentationContext_ can rotate and set a new basis coordinate system.==
```

Comment:
@pjanck shall we remove this sentence?

Reply:
I would propose to delete the last sentence and work on a general guideline on how to use rotation in this case and how to use the attribute "WorldCoordinateSystem" (misleading anyway) at _IfcGeometricRepresentationContext_ also in the IFC4.4 time frame.

Reply:
Coming from the requirement, that a "global engineering" context should be "rotatable" within the "global geodetic" context, I would argue that the documentation should be clear, where the rotation parameters can be transmitted.

1. At the level of `IfcCoordinateOperation`
2. At the level of `IfcGeometricRepresentationContext`
3. At both levels

Current possibilities resulting from this PR are in my understanding:

`IfcGeometricRepresentationContext.WorldCoordinateSystem` can set an 3D offset + 3D rotation in Cartesian coordinates relative to the base coordinate system defined by something (= STH); together with

A. If STH is `IfcMapConversion` or `IfcMapConversionScaled`, there is an additional 3D offset + 1D rotation that sets the base coordinate system relative to the coordinate reference system's coordinate system.
B. If STH is `IfcRigidOperation`, there is an additional 3D offset that sets the base point relative to the coordinate reference system's origin (the coordinate axes are inherited / overwritten, depending on the type of `IfcCoordinateReferenceSystem`).

Option A follows option 3 from above, while option B follows option 2 from above.

Thus:

a. There is a certain degree of unparallelity in options A and B. Should we strive to make this more similar?
b. Looking at option B and assuming no changes to the content, I believe that a note like the sentence referenced by this GitHub comment is needed.
c. If the PR is left as is, then I believe that this would be needed now:

> work on a general guideline on how to use rotation in this case and how to use the attribute "WorldCoordinateSystem"

Reply:
Can the `IfcGeometricRepresentationContext.WorldCoordinateSystem` be deprecated?

Reply:
+1 from my side, also from a practical point of view - getting the different ways (or combinations of different ways) to establish the georeferencing correct is already too complex. In any case, the current definition of the WorldCoordinateSystem includes a note that describes its usage as rigid operation. See [here](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcGeometricRepresentationContext.htm).

I see an unneccessary split of the functionality "offsetting and rotation" between two places, the new `IfcRigidOperation` and the `IfcGeometricRepresentationContext.WorldCoordinateSystem` - wouldn't a option D be more precise:

deprecate WorldCoordinateSystem and add rotation to IfcRigidOperation?

the deprecation note at WorldCoordinateSystem would state that IfcRigidOperation should be used instead. And the offset of "project base point" and "survey point" should be handled by IfcSite.ObjectPlacement as long as the well established rule - max. 1 instance of IfcSite is maintained. (this at least is what I see in 99,99% of all IFC files).

Reply:
> deprecate WorldCoordinateSystem and add rotation to IfcRigidOperation?

I can definitely live with that as well. 

1. For the sake of symmetry, should the rotation be only allowed around the z axis as with the `IfcMapConversion`?
2. The documentation of `WorldCoordinateSystem` shall be revised in that case.
- The deprecation note should state that `IfcCoordinateOperation` shall be used instead.
- `However it is possible to offset the geographic reference point from the local zero point.` shall be deleted.
3. Looking at

> should be handled by IfcSite.ObjectPlacement 

this does not account for structures residing next to `IfcSite`, that will probably become more popular in infrastructure - see [example here](https://github.com/buildingSMART/IFC4.x-IF/blob/main/tests/TR01/README.md). We need to be more precise with the formulation.